src/components/Menu.js

```diff
@@ -35,7 +35,14 @@ const renderMenu = menuItem => {
               <p>
                 <b>Project Lockdown</b> uses a variety of sources to compile all the information showcased. We make sure to only consider sources that are reliable and verifiable.
                 You can check all the sources employed <a target="_blank" href="https://www.TIOF.Click/LockdownData">here</a>: 
-                <ul>
+                <ul class="ld-sources">
+                  <li>
+                    <a target="_blank" href="https://www.travel-advisory.info/">Travel-advisory API</a>
```

Comment:
When using target _blank we should add `rel="noopener noreferrer"` for security. See: https://pointjupiter.com/what-noopener-noreferrer-nofollow-explained/ (no need for nofollow).

Reply:
Done :)